nexus/src/external_api/views.rs

```diff
@@ -81,8 +81,22 @@ pub struct Image {
     #[serde(flatten)]
```

Comment:
Would it make sense to separate this out into `GlobalImage` and `ProjectImage` like we've done elsewhere?  Then clients can get that type-safety too (e.g., knowing that a `ProjectImage` always has a `project_id`).

Reply:
I like it, done in bd825ded